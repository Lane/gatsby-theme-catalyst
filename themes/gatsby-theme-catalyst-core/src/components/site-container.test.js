import React from "react"
import renderer from "react-test-renderer"
import SiteContainer from "./site-container"

describe("Header", () => {
  it("renders correctly", () => {

    const props = {
        children: 'test'
    }
    
    const tree = renderer
      .create(<SiteContainer {...props} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})