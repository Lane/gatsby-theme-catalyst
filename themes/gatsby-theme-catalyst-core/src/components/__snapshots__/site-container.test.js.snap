// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="css-155ux7s-SiteContainer"
>
  test
</div>
`;
